src/components/pages/observation/create/form/index.tsx

```diff
@@ -110,10 +130,66 @@ export default function ObservationCreateForm({ speciesGroups, languages }) {
       facts: {},
       userGroupId: currentGroup.id > 0 ? [currentGroup.id.toString()] : [],
       resources: [],
-      terms: true
+      terms: true,
+
+      customFields: customFieldList?.map(
```

Comment:
I think this might more cleaner if we extract this into seprate function or memoized one if frequent renders